res/css/components/views/location/_ZoomButtons.scss

```diff
@@ -16,30 +16,30 @@ limitations under the License.
 
 .mx_ZoomButtons {
     position: absolute;
-    bottom: $spacing-32;
-    right: $spacing-24;
-}
-
-.mx_ZoomButtons_button {
-    @mixin ButtonResetDefault;
-
-    margin-top: $spacing-8;
-    border-radius: 4px;
+    inset-block-end: $spacing-32;
```

Comment:
Can we stick to `bottom` and `right` here? There is no text where buttons should be be aligned to depending on the text direction. Other elements on the view are using `top`, `left`… as well. Means if a user has a different text direction the layout may break.

Reply:
That makes sense. They could be changed at the same time in order to make the whole UI compatible with RTL layout.

Reply:
The commit was dropped.